pkgs/tools/filesystems/bees/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, makeWrapper, fetchFromGitHub, btrfs-progs, coreutils, pythonPackages, utillinux }:
+
+let
+
+  version = "0.6";
+  sha256 = "13vgbvpddnfyxjl7vnk0br9wbd8alv52935rxzvr8li8a5pl9qhx";
+
+  bees = stdenv.mkDerivation rec {
+    name = "bees-${version}";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "Zygo";
+      repo = "bees";
+      rev = "v${version}";
+      inherit sha256;
+    };
+
+    buildInputs = [
+      btrfs-progs               # for btrfs/ioctl.h
+      utillinux                 # for uuid.h
+      pythonPackages.markdown   # documentation build
```

Comment:
Use `nativeBuildInputs` for things that are only used during the build. The documentation building dependencies for example.